internal/command/commands/service.go

```diff
@@ -346,11 +391,9 @@ func (s *Service) Run(args []string) int {
 
 // Help : returns an explanatory string
 func (s *Service) Help() string {
-	if s.Create {
+	if s.Release {
 		return commandHelper("create", "service", "", []Options{
-			{Flag: "--file", Description: "json file to read service definition"},
-			{Flag: "--name", Description: "name of the service (required if using --rebuild)"},
-			{Flag: "--version", Description: "version of the service (required if using --rebuild)"},
+			{Flag: "--path, -p", Description: "path to directory containing service definition and provisioning config"},
```

Comment:
why are these flags removed?
What if image creation failed when releasing, we need a way to retrigger the image creation